docs/services/ihatemoney.md

```diff
@@ -0,0 +1,46 @@
+# ihatemoney
+
+[ihatemoney](https://github.com/spiral-project/ihatemoney) is a self-hosted shared budget manager, that this playbook can install, powered by the [ansible-role-ihatemoney](https://github.com/IUCCA/ansible-role-ihatemoney) Ansible role.
+
+
+## Dependencies
+
+This service requires the following other services:
+- a [Postgres](postgres.md) database
+- a [Traefik](traefik.md) reverse-proxy server
+
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# ihatemoney                                                              #
+#                                                                      #
+########################################################################
+
+ihatemoney_enabled: true
+
+# To enable the Admin dashboard, first generate a hashed password with:
+# docker run -it --rm --entrypoint ihatemoney ihatemoney/ihatemoney generate_password_hash
+#ihatemoney_admin_password:
+ihatemoney_environment_variable_public: false
```

Comment:
It's a bit unclear what `ihatemoney_environment_variable_public` actually does.

Should it be set to `true` at some point later?

Reply:
If your instance is meant to be open for everyone to freely create projects, you would set it to true.
If your instance is intended for private use, and you want to restrict project creation to users with the admin password, you would set it to false (or simply not set it, as false is the default).

I have adapted the documentation a little, now it should be clearer